test/tz_stub.py

```diff
@@ -7,7 +7,7 @@
 import pytest
 
 try:
-    import zoneinfo  # type: ignore[import]
```

Comment:
Had to made this change for mypy to run.

Reply:
Or mb nervermind. It might be caused by my local mypy config. Reverted and rerunning CI again